src/pages/iou/MoneyRequestAmountForm.tsx

```diff
@@ -51,7 +51,7 @@ type MoneyRequestAmountFormProps = Omit<MoneyRequestAmountInputProps, 'shouldSho
     chatReportID?: string;
 };
 
-const isAmountInvalid = (amount: string) => !amount.length || parseFloat(amount) < 0.01;
+const isAmountInvalid = (amount: string) => !amount.length || parseFloat(amount) < 0;
```

Comment:
Why does this need to be changed? And also why are we parsing the amount as a float, don't we intentionally keep all amounts as ints for amounts?